File: pset1/mario/mario.c
#include<cs50.h>
#include<stdio.h>

int main(void)
{
    //initializing all the variables used
    int n, i, j;
    do
    {
        //asking and accepting the height for the pyramid from the user
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8); //checking for the number to non negative and no more greater than 8
    
    //Loop for forming the rows of the pyramid
    for (i = 0; i < n; i++)
    {   
        //Loop for space before the right aligned pyramid
        for (j = 0; j < n - i - 1; j++)
        {
            printf(" ");
        }
        // Loop for right aligned pyramid
        for (j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        // space between the left aligned and right aligned pyrmaid to give a complete pyramid
        printf("  ");
        // Loop for left aligned pyramid
        for (j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        //for entering new line
        printf("\n");
    }
}

File: sentimental/mario/less/mario.py
from cs50 import get_int

# asking and accepting the height for the pyramid from the user
while True:
    h = get_int("Height: \n")
    if h < 1 or h > 8:
        h = get_int('height: ')
    if h >= 1 or h <= 8:
        break

# Loop for left side pyramid
for i in range(h):
    print((h - 1 - i) * " ", end="")
    print((i + 1) * "#")
